### DISCUSSION TITLE
how to change a custom color for steps?

### DISCUSSION THREAD

**Discussion by dominicdev** [saadeghi/daisyui#992] [Q&A] [answered]
I can we change custom color for steps? direcly in the classname? cause if I gonna add colors in config will messup the whole project.. thanks

**Comment by rhscjohn-dev** [comment, marked as answer]
One way would be to add custom style(s) for the daisyui class: step to your tawilwind css file.

Here is a [Tailwind Play Example](https://play.tailwindcss.com/WkPwpfhVrE) for the Step component with a custom style color.   See line 5 and click on the CSS tab to see the custom styles that were added to tailwind.

**Comment by saadeghi** [comment]
Another quick way is using class names like this
```html
<ul class="steps">
  <li class="step step-info">info</li>
  <li class="step after:!bg-pink-400">pink-400</li>
  <li class="step after:!bg-[#f47f21]">custom orange</li>
</ul>
```
https://play.tailwindcss.com/agi0n72qd6

**Reply by dominicdev** [reply]
thanks

**Reply by Anibalfermandois** [reply]
This worked great!
